docs/services.md

```diff
@@ -1,114 +1,185 @@
-These are the core services which we provide across our platform.
+# Services
+
+ACP (Application Container Platform) is a series of services that the Home Office recommends developers use to develop and host Home Office applications. This page provides an overview of how ACP uses each service.
+
+To use ACP, you require knowledge of the following services:
+
+  * Docker: a software tool designer that allows you to create, deploy and run applications using containers
+  * Kubernetes: an open-source platform for automating deployment, scarling, and operation of application containers across clusters of hosts
+  * GitHub and GitLab: repositories which store open-source and private code respectively
+  * Drone: a CI (Continuous Integration) tool for building and deploying applications
+  * Quay: an open-source container repository
+  * Amazon Elastic Container Registry (ECR): a container registry for storing, managing, and deploying private Docker container images.
+  * Artifactory: an internal repository to store artifacts (including binaries and dependencies) to
+  * Keycloak: an open-source identity and access-management tool
+  * Sysdig: an alert monitoring tool for containers, Kubernetes and cloud-based services
+
+Where a service requires a bespoke or custom configuration to use ACP, we will provide details of the prerequisites. For example, for instructions on how to set up your services for the ACP induction, see [Developer Onboarding][dev_onboarding].
+
+Any other information on individual services is outside the scope of our documentation. For information on how to achieve configurations on the service side, we recommend referring to the service vendor’s documentation.    
+
+
 
 ## VPN
-Many of our services are behind a VPN (using [OpenVPN](https://wiki.archlinux.org/index.php/OpenVPN) protocol) for security reasons. We have many different roles and profiles for accessing different environments. These profiles are all found at [Access ACP](https://access-acp.digital.homeoffice.gov.uk).
-You can download VPN profiles for the environments you have access to. New profiles can be set up to connect through to things like project specific AWS accounts.
-Each profile will expire after a certain amount of time (8-72 hours), users will have to download a new profile once their certificates which are baked into the openvpn profiles expire.
+
+
+For security reasons, ACP services run behind a VPN using the OpenVPN protocol.
+
+Download VPN Profiles from the [ACP VPN][acp_vpn]. The kube-platform profile provides you with access to ACP services.
+
+Note that in addition to a VPN profile, some environments require access rights. If you want to also manage production namespaces, raise a support ticket.
```

Comment:
This could lead to users approaching us rather than their project admins when they need tokens to access their production namespaces. What we really want to say is that another VPN profile may be needed to access to the production environment for which a ticket id needed.